apps/server/src/modules/deletion/api/uc/deletion-request.uc.ts

```diff
@@ -65,53 +65,45 @@ export class DeletionRequestUc implements IEventHandler<DataDeletedEvent> {
 
 	async createDeletionRequest(deletionRequest: DeletionRequestBodyProps): Promise<DeletionRequestResponse> {
 		this.logger.debug({ action: 'createDeletionRequest', deletionRequest });
+		const minutes =
+			deletionRequest.deleteAfterMinutes ?? this.configService.get<number>('ADMIN_API__DELETION_DELETE_AFTER_MINUTES');
+		const deleteAfter = new Date();
```

Comment:
for readability I think the type of `deleteAfter` is at first glance unclear to me.
maybe setting the type during declaration or changing the name to `deleteAfterMinute` / `deleteAfterDate` / `deleteAfterTime` ?